docs/user/aws/iam.md

```diff
@@ -0,0 +1,45 @@
+# IAM User
+
+In a new AWS account, you are provided with a “root” account. This account is based on the email address which created 
```

Comment:
nit: "“root” account" -> "root user" (to match Amazon's wording [here][1]).

[1]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html

Reply:
Fixed. Thanks!